java/ext/javadoctest.py

```diff
@@ -47,6 +47,10 @@ def compile(
     ) -> Any:
         # go to project that contains all your arrow maven dependencies
         path_arrow_project = pathlib.Path(__file__).parent.parent / "source" / "demo"
+        # delete classpath java dependencies files used
```

Comment:
What is the main reason to add this for Arrow Java version upgrade?

Reply:
Hi @vibhatha this change was for testing purpose, let me update with final changes